script.js

```diff
@@ -4,33 +4,209 @@ const logo = document.querySelector('.logo');
 const navMenu = document.querySelector('.nav-menu');
 const hamburger = document.querySelector('.hamburger');
 const navItem = document.querySelectorAll('.nav-item');
+const modal = document.querySelector('.modal');
+const cards = document.querySelector('.card-works');
+const wrapper = document.querySelector('.wrapper');
+const overFlow = document.querySelector('*');
 
-const openModal = () => {
+const openMenu = () => {
   logo.classList.toggle('active');
   navMenu.classList.remove('desktop');
   navMenu.classList.toggle('active');
   hamburger.classList.toggle('active');
 };
 
-const closeModal = () => {
+const closeMenu = () => {
   navMenu.classList.add('desktop');
   logo.classList.remove('active');
   navMenu.classList.remove('active');
   hamburger.classList.remove('active');
 };
 
 hamburger.addEventListener('click', () => {
-  openModal();
+  openMenu();
 });
 
 navItem.forEach((n) => {
   n.addEventListener('click', () => {
-    closeModal();
+    closeMenu();
   });
 });
 
 document.addEventListener('keydown', (e) => {
   if (e.key === 'Escape') {
-    closeModal();
+    closeMenu();
   }
 });
+
+const projects = [
+  {
+    title: 'Multi-Post Stories',
+    technologies: ['html', 'css', 'ruby on rails'],
+    imageURL: './assets/images/content/Img-Placeholder.svg',
+    content:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    description:
```

Comment:
- [Optional] **Recommended**  I can see you have put in a lot of work trying to implement the required changes from the previous reviewers, but it's not perfect yet. Please just as the previous reviewers have requested try to make your popup details align with that of the Figma design. (on the button)

##### Figma design 
![Screenshot from 2022-07-21 17-41-15](https://user-images.githubusercontent.com/58771507/180271331-5f46694f-a104-43f3-8daf-1d4ec7d5ba46.png)


##### YOur design 
![Screenshot from 2022-07-21 17-49-07](https://user-images.githubusercontent.com/58771507/180271280-f5efc99f-c05a-4fec-8c9b-fb6d291f101a.png)

Reply:
Thank you @Ginohmk for your feedback, I will modify it.